pkg/controller/kmeshsecurity/security_manager.go

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright 2024 The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kmeshsecurity
+
+import (
+	"sync"
+	"time"
+
+	"container/heap"
+
+	"istio.io/istio/pkg/security"
+	"kmesh.net/kmesh/pkg/controller/workload"
+	"kmesh.net/kmesh/pkg/logger"
+)
+	
+type Certs struct {
```

Comment:
```Certs``` can be renamed to ```Cert```? As it represents a single certificate

Reply:
OK